igvc_gazebo/nodes/scan_to_pointcloud/scan_to_pointcloud.cpp

```diff
@@ -1,5 +1,6 @@
+#include "scan_to_pointcloud.h"
+
 #include <laser_geometry/laser_geometry.h>
-#include <parameter_assertions/assertions.h>
```

Comment:
Why did you remove the assertions header? Don't you use it to get the `min_dist`,  `neighbor_dist`, and `offset` parameters?